en/docs/choreo-concepts/third-party-service.md

```diff
@@ -0,0 +1,10 @@
+# Third Party Service
```

Comment:
In the Concepts section, we typically include terms or ideas that have a specific meaning or purpose within the context of Choreo. Since third-party services are not specific to Choreo, I feel it would be more appropriate to include this information within the main content, rather than in the Concepts section. WDYT?